packages/docs/.storybook/main.js

```diff
@@ -8,13 +9,15 @@ const config = {
     '@storybook/addon-essentials',
     '@storybook/addon-a11y',
     '@storybook/preset-create-react-app',
-    '@storybook/addon-storysource',
   ],
   staticDirs: ['../public'],
   webpackFinal: async (config) => {
     config.resolve = config.resolve || {}
     config.resolve.alias = {
       ...config.resolve.alias,
+      'msw/native': require.resolve(
+        path.resolve(__dirname, '../../../node_modules/msw/lib/native/index.mjs')
+      ),
```

Comment:
this is necessary because the export conditions aren't finding this package, likely because it lacks a browser export

Reply:
In my case the error is with `msw/node` (using storybook with vite builder) in this line: https://github.com/mswjs/msw-storybook-addon/blob/0ff898a0272df0a6eeebe9faa09b3690509a8b53/packages/msw-addon/src/mswDecorator.ts#L56

but I'd find it hard to sell that consumers of this plugin (or really, consumers wanting to use storybook + jest + jsdom + msw) need to do this 👇 
```
      'msw/native': require.resolve(
        path.resolve(__dirname, '../../../node_modules/msw/lib/native/index.mjs')
      ),
      'msw/node': require.resolve(
        path.resolve(__dirname, '../../../node_modules/msw/lib/node/index.mjs')
      ),
```

But I also know there is a lot of push back about those conditional `exports` 😞

- https://github.com/mswjs/msw/issues/1877
- https://github.com/mswjs/msw/issues/1809
- https://github.com/mswjs/msw/issues/1787
- https://github.com/mswjs/msw/issues/1786